src/elements/container/sticky-bar/sticky-bar.visual.spec.ts

```diff
@@ -33,17 +33,25 @@ describe(`sbb-sticky-bar`, () => {
   const containerContent = (color?: string): TemplateResult => html`
     <sbb-title level="4" ?negative=${!!color && isDark(color)}>Example title</sbb-title>
     <p class="sbb-text-s">The container component will give its content the correct spacing.</p>
-    <sbb-secondary-button style="margin-block-end: 0.75rem;" size="m">
+    <sbb-secondary-button
+      style="margin-block-end: 0.75rem;"
+      size="m"
+      ?negative=${!!color && isDark(color)}
```

Comment:
For the button it's not necessary as one can use the normal variant also in dark context. but it's also ok to use the negative variant :-) so let's keep the change